### DISCUSSION TITLE
"Include Once" directive

### DISCUSSION THREAD

**Discussion by ReallyWeirdCat** [wiremod/wire#2878] [Suggestions]
A directive that could prevent the file from being included a second time if it was already included in the project before during the iteration. This is needed so that both files could have visibility of each other's variables and methods in the editor

**Comment by Denneisk** [comment]
Maybe we'll get the C-style
```c
#ifndef LIB_NAME
#define LIB_NAME
...
```

**Comment by vurvdev** [comment]
I don't see a directive for this happening. Above preprocessor solution maybe. But then again I want to decrease reliance on the preprocessor. You could do this pretty easily with just a `@persist` variable since `#include` works at runtime in E2.
